ONSdigital/eq-submitter
# Eq 16 submitter foundation

**_What**_

A Simple app that receives messages from RabbitMQ Cluster and places them in a rabbitMQ bucket.

**_How to test**_
1) Log into the RabbitMQ Console for the environment you wish to test.
2) Verify the queue exists
3) Place a message on the queue using the RabbitMQ Admin
4) Verify that the message appears in your environments S3 Bucket

**_Who can test**_

Anybody except @weapdiv_david
